### DISCUSSION TITLE
Shrink Network to Avoid Overfitting

### DISCUSSION THREAD

**Discussion by jhogan7** [MIC-DKFZ/nnUNet#958] [Q&A]
Yes. Thank you for developing this model, it is exactly what it aims to be: user-friendly and effective.

From reading other discussions, you mentioned that if the green line/evaluation metric does not decrease we do not have to worry about overfitting. What is the rationale behind not worrying about overfitting if the training loss is decreasing while the validation loss does not change (see results for 3D_full_res below)? Also, what is the advantage of running all 1000 epochs in this case where it appears to be overfitting? 

![image](https://user-images.githubusercontent.com/54959376/156607474-34bc957d-4231-49e2-b177-50da05588271.png)

Furthermore, if we wanted to shrink the network have you had experience with the following? 
    (1) changing self.unet_base_num_features from 32 to 16 or 8
    (2) changing dropout probability from 0 to 0.5
    (3) changing conv_per_stage from 2 to 1

It would be great to get any thoughts you have on the issue, thank you for your time!

**Comment by FabianIsensee** [comment]
Can you please provide some information?
